### DISCUSSION TITLE
2.17 Dev Branch

### DISCUSSION THREAD

**Discussion by ThioJoe** [ThioJoe/YT-Spammer-Purge#793] [Announcements]
I've added a new ["Dev" branch](https://github.com/ThioJoe/YT-Spammer-Purge/tree/2.17-Dev) and included a bunch of the pull requests that looked promising but I haven't gotten around to fully testing quite yet. So if you want to test it feel free and report any issues.

There won't be any actual releases for the Dev branch, so you'll have to run it with python.

**Comment by DinhHuy2010** [comment]
yes

**Comment by DinhHuy2010** [comment]
I downloaded the version

![Screenshot 2022-04-11 142300](https://user-images.githubusercontent.com/95196459/162685527-2815192f-4df0-41b1-b764-a570272be8cc.png)
![Screenshot 2022-04-11 142423](https://user-images.githubusercontent.com/95196459/162685755-d6d62c67-7c2d-4b44-b558-802e7902fcb8.png)
![Screenshot 2022-04-11 142514](https://user-images.githubusercontent.com/95196459/162685944-acf96ed3-0c73-42e4-bd98-05f24fa276b3.png)

**Comment by Tyler887** [comment]
Can someone answer https://github.com/ThioJoe/YT-Spammer-Purge/discussions/814? I need to know how to run it as an exe.

**Reply by Firecul** [reply]
[Kurdakov](https://github.com/Kurdakov) Has already answered you.

(Anyone that can answer prob checks this repo often or has notifications on, no need to ask in other posts.)
